### PR TITLE
Fix various eslint errors

### PR DESCRIPTION
**Pull request by esoergel**
This file is terrifying, I'm dreading what git blame is now going to say I'm responsible for.  First commit is whitespace only, second commit is logical.
:tropical_fish: :heavy_multiplication_x: :whale:

### CODE REVIEW

File: corehq/apps/locations/static/locations/ko/location_drilldown.async.js [hunk 1]
```diff
@@ -1,221 +1,223 @@
...
-  this.selected_location = ko.computed(function() {
-      for (var i = this.selected_path().length - 1; i >= 0; i--) {
-        var loc = this.selected_path()[i];
-        if (loc.selected_is_valid()) {
-          return loc.selected_child();
+    var model = this;
+
+    this.default_caption = default_caption || 'All';
+    this.auto_drill = (_.isBoolean(auto_drill) ? auto_drill : true);
+    this.loc_filter = loc_filter || function(loc) { return true; };
```
Comment (dimagimon) [inline corehq/apps/locations/static/locations/ko/location_drilldown.async.js]: 'loc' is defined but never used (no-unused-vars)

File: corehq/apps/locations/static/locations/ko/location_drilldown.async.js [hunk 2]
```diff
@@ -1,221 +1,223 @@
...
+    this.get_child = function(i) {
+        return this.children()[i + 1];
+    };
+
+    this.load = function(data) {
+        this.name(data.name);
+        this.type(data.location_type);
+        this.uuid(data.uuid);
+        this.can_edit(_.isBoolean(data.can_edit) ? data.can_edit : true);
+        if (!!data.children) {
```
Comment (dimagimon) [inline corehq/apps/locations/static/locations/ko/location_drilldown.async.js]: Redundant double negation. (no-extra-boolean-cast)

**Comment by esoergel** [comment]
@millerdev

File: corehq/apps/locations/static/locations/ko/location_drilldown.async.js (hunk 1 above)
Comment (esoergel) [inline corehq/apps/locations/static/locations/ko/location_drilldown.async.js]: I actually prefer to leave that in.

Comment (dimagimon) [inline corehq/apps/locations/static/locations/ko/location_drilldown.async.js]: 'loc' is defined but never used (no-unused-vars)

**Comment by benrudolph** [comment]
i did this once on a touchforms file. it's quite the incentive to refactor it haah

**Pull request merged**